src/SourceBuild/content/eng/pipelines/ci-lite.yml

```diff
@@ -0,0 +1,21 @@
+# This is the dotnet/dotnet-lite pipeline that is triggered by all PRs and pushes to main.
+
+trigger:
+  batch: true
+  branches:
+    include:
+    - main
+
+pr:
+  branches:
+    include:
+    - '*'
```

Comment:
We typically scope this down to     
- main
- release/*

Reply:
Fixed in https://github.com/dotnet/installer/pull/17370/commits/bf697aa5cab242f80871e0db905b3d85ba24dedf